### PR TITLE
Whitelist APIs for generic Pub-Sub

### PR DESCRIPTION
**Pull request by tomusdrw**
Ping @kaikun213

### CODE REVIEW

File: parity/rpc_apis.rs [hunk 1]
```diff
@@ -566,6 +570,13 @@ impl ApiSet {
 				public_list.insert(Api::Personal);
 				public_list
 			},
+			ApiSet::PubSub => vec![
```
Comment (maciejhirsz) [inline parity/rpc_apis.rs]: can be just a slice or a fixed array? (Same for line 539)

Comment (maciejhirsz) [inline parity/rpc_apis.rs]: Small grumble: could be just a slice or a fixed array instead of a vec? Same for line 539.

Comment (tomusdrw) [inline parity/rpc_apis.rs]: You cannot consume elements from an array or slice afaict. `into_iter()` is implemented only for `&` and returns `IntoIterator<Item=&T>`, so we would need to clone the items.

Comment (rphmeier) [inline parity/rpc_apis.rs]: definitely not a bottleneck in startup -- but cloning N 1-byte enum values should be really quick.

Comment (tomusdrw) [inline parity/rpc_apis.rs]: Fixed.

**Pull request merged**